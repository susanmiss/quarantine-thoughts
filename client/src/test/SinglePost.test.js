import React from 'react';
import ReactDOM from 'react-dom'
import {shallow} from 'enzyme'; 
import renderer from 'react-test-renderer';
import SinglePost from '../SinglePost'


const setup = (props={}, state=null) => {
    const wrapper = shallow(<SinglePost {...props} />)
    if( state ) wrapper.setState(state);
    return wrapper;
  }

  const findByTestAttr = (wrapper, val) => {
    return wrapper.find(`[data-test="${val}"]`);
  }

describe("<SinglePost />", () => {


  it('renders correctly', () => {
      const tree = renderer
          .create(<SinglePost />)
          .toJSON();
      expect(tree).toMatchSnapshot();
  });
  let wrapper;
  beforeEach(() => wrapper = shallow(<SinglePost />));
  
  it('Renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
  })
  
  it('should render a <div />', () => { 
      expect(wrapper.find('div').length).toEqual(1);
  });


})