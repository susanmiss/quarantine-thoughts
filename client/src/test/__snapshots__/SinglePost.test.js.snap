// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SinglePost /> Renders correctly 1`] = `ShallowWrapper {}`;

exports[`<SinglePost /> renders correctly 1`] = `
<div
  className="container pb-5"
>
  <br />
  
</div>
`;
